src/plugins/intel_cpu/tests/functional/subgraph_tests/src/conv_sum_broadcast.cpp

```diff
@@ -331,12 +332,14 @@ InputShape secondInp = {
             {1, 64, 8, 8},
             {1, 64, 8, 1}
         }
+    },
+    {{1, 64, 8, 8}, {{1, 64, 8, 8}}},
```

Comment:
Could you please preserve the format above?

Reply:
Done, thanks